lib/koans/18_genservers.ex

```diff
@@ -135,15 +135,11 @@ defmodule GenServers do
 
   koan "Our server works but it's pretty ugly to use; so lets use a cleaner interface" do
     Laptop.start_link("EL!73")
-    assert Laptop.unlock("EL!73") == ___
-  end
 
-  koan "Let's use the remaining functions in the external API" do
```

Comment:
❓ The Genserver and Agent koans were timing out when running tests. It appears to have something to do with processes "leaking" between examples. It's been awhile since I've worked in Elixir, so I wasn't able to debug that problem. Instead I opted to fix this by:
1. Using distinct names in the Agents koan.
2. Combining the problematic examples sin the GenServers koan.